README.md

```diff
@@ -1,9 +1,29 @@
 The ev3dev meta repo
 ====================
-This repository doesn't actually have any code. It hosts [downloads](https://github.com/ev3dev/ev3dev/releases),
-the ev3dev [wiki](https://github.com/ev3dev/ev3dev/wiki) and the [issues](https://github.com/ev3dev/ev3dev/issues) tracker.
 
-To learn more about ev3dev, check out our home page at <http://www.ev3dev.org>.
+This repository doesn't actually have any code. It hosts [downloads](https://github.com/ev3dev/ev3dev/releases) and the project-wide  [issues](https://github.com/ev3dev/ev3dev/issues) tracker.
```

Comment:
How about "SD card image downloads" instead of just "downloads"?

You could also stick "tracker" in the link with "issues"